### PR TITLE
Attribution Guidelines

### LINKED ISSUE #1328: Clarify TAG attribution policies
This relates to #1321 as well as #1311.

Currently, a reasonable person can draw different conclusions as to when and how TAG Security can be attributed. 

For example, whitepapers, public statements, project communications, STAG website blogs, and other blogs. 

- If a whitepaper is produced by TAG members or a TAG Working Group, when is it considered to be a TAG Publication?
- If a blog post is published on our website, how do we delineate between the opinion of the author and the advice of the TAG?
- If an opinion piece be written by a TAG Security member or leader is published on an external site, how can they reflect their TAG affiliation without appearing to represent the TAG?
- How can an advice document be published on an external site that _does_ come from the TAG?
- When interacting with projects as part of TAG duties, when is it appropriate to say that guidance is from the TAG? Similarly, how can a TAG leader best give recommendations without appearing to represent the TAG?
- etc.

### PR DESCRIPTION
**Pull request by eddie-knight**
This will close #1328.

### CODE REVIEW

File: governance/tag-attribution.md
Comment (JustinCappos) [inline governance/tag-attribution.md:34]: spelling

File: governance/tag-attribution.md
Comment (JustinCappos) [inline governance/tag-attribution.md:47]: can you add an example with multiple affiliations?

Comment (eddie-knight) [reply governance/tag-attribution.md:47]: Good idea! Applied.

**Review by anvega — CHANGES_REQUESTED** [review]
This proposed policy on TAG attribution seems redundant considering CNCF already differentiates between TAG, Member, and Community posts. Examples include:

* [TAG Post: Evaluating Your Supply Chain Security](https://www.cncf.io/blog/2021/05/14/evaluating-your-supply-chain-security)
* [Member Post: Secure Software Supply Chains ](https://www.cncf.io/blog/2021/10/27/secure-software-supply-chains-good-practices-at-scale)
* [Community Post: Introduction to the Cloud Native Security Controls Catalog](https://www.cncf.io/blog/2022/06/07/introduction-to-the-cloud-native-security-controls-catalog/)

Adding more layers of approval and attribution processes not only duplicates existing distinctions but also risks centralizing control over information. This can stifle individual contributions and diverse perspectives, potentially leading to a monopolization of thought. If someone has a counter perspective to something anyone says public, they should be able to express it too in a timely fashion without facing bureaucratic hurdles.

For example, the [Security Slam 2023](https://www.cncf.io/reports/lightning-round-at-security-slam-2023/) post attributed to TAG Security was never vetted through the suggested rigorous process. This inconsistency highlights the potential pitfalls of over-regulating attribution. Additionally, there are members who frequently flex their association with the TAG on social channels without any disclaimers, which undermines the argument for strict guidelines.

We should aim for clarity and flexibility, enabling members to leverage their affiliations responsibly without unnecessary bureaucratic obstacles.

**Comment by eddie-knight** [comment]
#1328 is in response to tandem requests for clarification around whitepaper attribution as well as the recent blog post discussion.

@anvega — as you noted before in a previous email thread, the recent blog post would not have raised any discussion at all if there was not a misunderstanding regarding attribution. This addition aims to alleviate that misunderstanding.

If other posts are effected by this in the future, it will be for the sake of ensuring that the advisory group is involved when issuing opinions and advice. Cases where the TAG is not being attributed as an author will not be affected by this addition.

**Comment by anvega** [comment]
What does white paper attribution have to do with this? If a paper is merged in here, it becomes TAG Security’s paper.

Just to be precise, "effected" should actually be "affected." I guess this proposed policy is already creating a ripple of confusion. Jokes aside, I'm still concerned about the implications. Adding these layers of approval and attribution processes could centralize control over information by chairs having to give permission for anyone to express their views or how they present those, potentially stifling individual contributions and diverse perspectives.

I don't know that I concur on labeling as misunderstanding what was deliberate action to disavow a communication through backchannels, but the policy would be to first check with the authors because 4 TAG Security people were involved in the authoring and review and 5 people including you present for the discussion of it.

File: governance/tag-attribution.md
Comment (anvega) [inline governance/tag-attribution.md:23]: My main contention is this:

> Blogs (On the TAG site or elsewhere)
> - The Blog must be directly connected to an initiative such as a whitepaper
>   or event.
> - If the blog contains any opinions, advice, or recommendations, it follows
>   all guidelines, protocols, and processes outlined in the [publications]
>   directory.

The collective of the people here nor its leadership structure have any inherence over what anyone says anywhere else.  As Justin said on the issue:

> If it's 'by someone who happens to have some affiliation, including TAG-Security', then I don't see why the group would need to review it. This might be done as a courtesy. However, for example, NYU doesn't review everything I say or do before I do so. We're all experts in the field and should be able to comment freely as ourselves without needing consensus.

**Review by anvega — CHANGES_REQUESTED** [review]

File: governance/tag-attribution.md
Comment (eddie-knight) [reply governance/tag-attribution.md:23]: There is a section regarding personal attribution, which covers the contention you mention. Can you highlight where it's not clear so that we can improve it?

Comment (anvega) [reply governance/tag-attribution.md:23]: `or elsewhere`

Comment (eddie-knight) [reply governance/tag-attribution.md:23]: The reason this is a universal statement is because this section contains `a list of works categories, and what will enable it to be attributed as a TAG publication`. 

The following section contains guidance for how to avoid confusion when posting personally, as in Justin's example.

`There is no restriction to this, but the following examples will aid in clarity to avoid mistaken attribution.`

**Comment by anvega** [comment]
While I understand the intent behind these guidelines, I find them misplaced. This is an open-source collective, and imposing stringent attribution requirements contradicts the principles of open-source collaboration. Being a core contributor or maintainer to a project should come with the liberty to discuss and promote the project. TAG Security doesn’t own people, and delineating how members present themselves is not in line with open-source ethos.

The control suggested here leans more towards arbitrary authority rather than technical and common sense. Historically, this hasn’t been an issue. The recent blog post was clarified with a simple comma in the byline, without needing extensive new guidelines. Despite attempts to claim guidelines weren't followed, no one is misattributing anything.

We should trust our members to manage their affiliations responsibly without bureaucratic oversight. We should foster collaboration and knowledge dissemination, not policing.

What’s the point of this? Maybe we can work backward from the intent here. Is it to police these rules? It’s better to reason with people when someone says something others can’t get on board with and request an amend or clarification, rather than to impose strict compliance.

Open source thrives on communication and collaboration, not rigid rules.

**Comment by JustinCappos** [comment]
No offense to anyone involved, but my sense is we may have passed the point of useful discourse on this issue.  

I propose that we merge this with a caveat that these are guidelines meant to be followed.  I'm also open to someone else chiming in with a way to close the issue and move on to other things.  :)

**Comment by anvega** [comment]
Sure, there will be situations where the presumption of sole authorship arises, like the recent instance. To demonstrate that guidelines were followed, should the blog piece be amended to include a statement like:

`This work was in collaboration with three other TAG members and signed off by the CTO, Executive Director, and Head of PR for the CNCF.`

Or where should that go?

This would cover the following:

> Attribution Qualifier
> It is preferable, but not required, that personally attributed works include a qualifying statement if the CNCF or the TAG is mentioned in the author byline or article content.
> 
> The author is a leader in CNCF's Technical Advisory Group for Security, but this work is the sole opinion of the author and does not represent any stance from CNCF or the TAG.

**Comment by matthewflannery** [comment]
> No offense to anyone involved, but my sense is we may have passed the point of useful discourse on this issue.
> 
> I propose that we merge this with a caveat that these are guidelines meant to be followed. I'm also open to someone else chiming in with a way to close the issue and move on to other things. :)

I feel like the best path is closing this issue by not merging it :D This implies mistrust, rather than trusting people won't misattibuted or misrepresent the group which no one ever has to my knowledge?

Considering after all that this _is_ open source software we are effectively talking about, shouldnt the goal be to encourage people to participate rather than creating rules for the sake of rules? It's hard enough for people to find time in their busy lives to volunteer expertise.

@eddie-knight is there any major benefit to this otherwise? Is CNCF or TAG Security at risk by not merging this in?

File: governance/tag-attribution.md
Comment (mnm678) [inline governance/tag-attribution.md:17]: Does this mean that a presentation would have to be approved every time someone gives a shout out to TAG Security? For example in the lead-up to CloudNativeSecurityCon I gave several explanations of what the TAG does, etc, which I would not have been able to do if it required an approval process.

**Comment by mnm678** [comment]
Another possible path forward would be to merge a more general form of this guidance that describes the intention of the policy (what's in the first paragraph), without setting specific requirements for types of publications, leaving us the flexibility to use our best judgement.

**Comment by anvega** [comment]
A general form of the guidance already exists in the Code of Conduct, which covers what constitutes work that belongs to the group. The spirit of what is proposed here is contradictory to that.

> Work performed within this group, either finalized or in draft, is to be used in accordance with the group Mission and Charter, the open source license, and to be used for the equal benefit of all members of the community. Further information on the use of work may be found in Security Reviews: Outcome.

Moreover, basing these guidelines on publication checklists, which I wrote for ebook material, will add toil to individuals giving presentations and other actitivies just as @mnm678 mentions.

File: governance/tag-attribution.md
Comment (eddie-knight) [reply governance/tag-attribution.md:17]: We could modify the second point below to use a type of "lazy consensus" language, but I imagine that yes— we would want to have some reasonable ability for leads to ensure that content is up to date before it is presented to the public as representative of the group.

For example, if @sublimino does a presentation about the TAG (as he often does), we should have a process that he can easily follow to help him share the latest from the group.

But if @sublimino was doing a presentation about his experience in the TAG (not representing the group) then it wouldn't benefit from the additional support layer.

**Pull request closed without merging**
